Clean Swift/Scene.xctemplate/UICollectionViewController/___FILEBASENAME___Interactor.swift

```diff
@@ -14,15 +14,15 @@ protocol ___FILEBASENAMEASIDENTIFIER___InteractorOutput {
 
 class ___FILEBASENAMEASIDENTIFIER___Interactor: ___FILEBASENAMEASIDENTIFIER___ViewControllerOutput {
     var output: ___FILEBASENAMEASIDENTIFIER___InteractorOutput!
-    var worker: ___FILEBASENAMEASIDENTIFIER___Worker!
```

Comment:
Understood. I've updated the naming to WorkerProtocol. 

As for worker->store variable name, let's switch it back to worker? The thing is that when you implement workerprotocol, the class is named store..wouldn't it be confusing?